src/magic_folder/scripts/magic_folder_cli.py

```diff
@@ -1059,7 +1061,7 @@ def do_magic_folder(options):
     try:
         return f(so)
     except Exception as e:
-        print("Error: %s" % (e,), file=options.stderr)
+        print(u"Error: {}".format(e), file=options.stderr)
```

Comment:
I did get tricked into *starting* to change every string literal in the whole project before I realized that was a trap, though...